.github/native-images.yaml

```diff
@@ -6,8 +6,10 @@ versions:
   - 20.3.2-java11
   - 21.2.0-java11
   - 21.2.0-java16
-  - 21.3.0-java11
-  - 21.3.0-java17
```

Comment:
Shouldn't we keep building `21.3` images for a while?

Reply:
Considering that 21.3.1 is a CPU update I don't see why anyone willing to use 21.3.0 would mind using an unpatched builder image :)

To my understanding the 21.3.0 builder image will still be available, it will just not rebuilt when there is an update on the underlying layers.

No strong opinion though, I guess it doesn't hurt much keeping it around (updated) for a while.

Reply:
It will still be used by a ton of people so IMHO we should continue updating it. We can remove `20.3*` and `21.2.0` instead IMHO. That will have the same effect and be less harmful.

Reply:
I have opened https://github.com/quarkusio/quarkus-images/pull/184 for removing `20.3.x`.

Reply:
I restore them.